.travis.yml

```diff
@@ -49,7 +49,9 @@ script:
     - mypy src --ignore-missing-imports
     - if [[ "$FULL_INSTALL" == "1" ]]; then
         jupyter --version;
-        source docs/notebooks/run_notebooks.sh;
+        chmod +x docs/notebooks/run_notebooks.sh;
```

Comment:
Rename `run_notebooks.sh` do something more descriptive. (e.g. `select_notebooks_to_execute`)

Reply:
I will refactor the filename.